guacamole/src/main/webapp/app/manage/controllers/manageUserController.js

```diff
@@ -918,6 +961,28 @@ angular.module('manage').controller('manageUserController', ['$scope', '$injecto
             else
                 removeConnectionGroupPermission(identifier);
 
+        },
+
+        /**
+         * Notifies the controller that a change has been made to the given
+         * sharing profile permission for the user being edited. This only
+         * applies to READ permissions.
+         *
+         * @param {String} identifier
+         *     The identifier of the sharing profile affected by the changed
+         *     permission.
+         */
+        sharingProfilePermissionChanged : function sharingProfilePermissionChanged(identifier) {
+
+            // Determine current permission setting
+            var value = $scope.permissionFlags.sharingProfilePermissions.READ[identifier];
```

Comment:
OK - change made via f3a201a. I went ahead and updated the other four cases where this naming was used within the same controller.